app/components/validated-input/component.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+const {
+    isEmpty,
+    computed,
+    defineProperty,
+} = Ember;
+
+import Validations from '../../validators/preprint-form-validator';
```

Comment:
Why does your general component (display logic for data) import a validator for one very specific use case?

Shouldn't the model you pass in be responsible for telling it what fields are valid?